packages/turf-concave/index.js

```diff
@@ -51,23 +53,32 @@ module.exports = function (points, maxEdge, units) {
 
     if (tinPolys.features.length < 1) throw new Error('too few polygons found to compute concave hull');
 
-    return merge(tinPolys.features);
+    // merge the adjacent triangles
+    var dissolved = dissolve(tinPolys.features);
+    // geojson-dissolve always returns a MultiPolygon
+    if (dissolved.coordinates.length === 1) {
+        dissolved.coordinates = dissolved.coordinates[0];
+        dissolved.type = 'Polygon';
+    }
+    return feature(dissolved);
 };
 
 /**
- * Merges/Unifies all the features in a single polygon
+ * Removes duplicated points in a collection returning a new collection
  *
  * @private
- * @param {Array<Feature>} features to be merged
- * @returns {Feature<(Polygon|MultiPolygon)>} merged result
+ * @param {FeatureCollection<Point>} points to be cleaned
+ * @returns {FeatureCollection<Point>} cleaned set of points
  */
-function merge(features) {
-    var merged = clone(features[0]);
-    merged.properties = {};
-
-    for (var i = 0, len = features.length; i < len; i++) {
-        var poly = features[i];
-        if (poly.geometry) merged = union(merged, poly);
-    }
-    return merged;
+function removeDuplicates(points) {
+    var cleaned = [];
+    var existing = {};
+    points.features.forEach(function (pt) {
```

Comment:
👍 This syntax looked so familiar that I thought I wrote this part :)

https://github.com/Turfjs/turf/blob/master/packages/turf-clean-coords/index.js#L52-L58